nixos/modules/services/audio/roon-server.nix

```diff
@@ -33,9 +47,7 @@ in {
       serviceConfig = {
         ExecStart = "${pkgs.roon-server}/opt/start.sh";
         LimitNOFILE = 8192;
-        DynamicUser = true;
-        SupplementaryGroups = "audio";
-        StateDirectory = name;
+        User = cfg.user;
```

Comment:
Don't forget `Group = cfg.group;`!

Reply:
Actually, I believe systemd sets the group to be the primary group of the user by default. But I also prefer having it set here